src/scripts/pokemons/PokemonFactory.ts

```diff
@@ -62,7 +62,7 @@ class PokemonFactory {
     public static routeDungeonTokens(route: number, region: GameConstants.Region): number {
         route = MapHelper.normalizeRoute(route, region);
 
-        const tokens = Math.max(1, 6 * Math.pow(this.routeLevel(route,region) / (3 / Math.round(1 + region() / 3)), 1.05));
+        const tokens = Math.max(1, 6 * Math.pow(this.routeLevel(route,region) / (3 / Math.round(1 + player.region() / 3)), 1.05));
```

Comment:
```suggestion
        const tokens = Math.max(1, 6 * Math.pow(this.routeLevel(route,region) / (3 / Math.round(1 + region / 3)), 1.05));
```

Reply:
Forgot it already defined region for a sec